playbooks/roles/ocp-config/templates/chrony.conf.j2

```diff
@@ -1,6 +1,9 @@
+{% if chronyconfig.content is defined %}
 {% for item in chronyconfig.content %}
 server {{ item.server }} {{ item.options }}
 {% endfor %}
+{% endif %}
+server {{ ansible_default_ipv4.address }} iburst
```

Comment:
You can use {{ ansible_host }} here.

Reply:
Good catch .... but we don't need this file anymore as it is now done by helpernode...
I'm gonna remove it (and another one) and run a new serie of test to validate.

Reply:
Cool

Reply:
done